### DISCUSSION TITLE
eip155 in proposal namespaces

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1939] [Developer Support]
I'm implementing WC v2 using universal-provider. From the documentation, I can see v2 now requires the chains to be specified in proposal namespaces. Our chain is an Ethereum sidechain with ID 2020, which syntax should I use in the namespace: eip155:2020 or <our_chain_name>:2020

Asked on Discord!

**Comment by ligi** [comment]
very likely eip155:2020 - https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md
is your chain PublicMint?

**Reply by pkhien95** [reply]
Our chain is not PublicMint, it's Ronin (https://docs.roninchain.com/docs/intro/mainnet). Is it still correct to use `eip155:2020`?
